FAQ.md

```diff
@@ -1,19 +1,37 @@
-# 🔮 FAQ
-
-1. ## Fonts don't seem to be correct
-
-   Add `@import url('https://fonts.googleapis.com/css?family=Source+Sans+Pro:300:400,600&display=swap');` to your applications CSS.
-
-2. ## Content-Security-Policy (CSP) problems?
-
-   You need to configure DEV env to allow
-
-   ```bash
-   style-src: 'unsafe-inline'
-   ```
-
-   And also might need to set `dangerously omit style nonce` to `true`.
-
-3. ## Some components won't work on old browsers (e.g. IE) at all
-
-   Add polyfills to your application.
+# 🔮 FAQ
+
+## 1. Fonts don't seem to be correct
+
+Add `@import url('https://fonts.googleapis.com/css?family=Source+Sans+Pro:300:400,600&display=swap');` to your applications CSS.
+
+## 2. Content-Security-Policy (CSP) problems?
+
+If strict content security policy is not required, you can simply use
+
+```bash
+style-src: 'unsafe-inline'
+```
+
+to allow the use of dynamic inline styling.
+
+### Strict content security policies
+
+Strict content security policy does not allow dynamic styling without extra safety measures. Styled components uses `__webpack_nonce__` to provide these measures.

```

Comment:
Not sure how styled components should be written, but at least this way is used in their website: "As you can see, styled-components"